### DISCUSSION TITLE
Silhouette for Play 2.9

### DISCUSSION THREAD

**Discussion by nemoo** [playframework/playframework#12028] [General]
I tried Play 2.9-RCX with the current Silhouette version "io.github.honeycomb-cheesecake" %% "play-silhouette" % "8.0.2".

```
[error] java.lang.RuntimeException: found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error] 
[error] 	* org.scala-lang.modules:scala-xml_2.13:2.2.0 (early-semver) is selected over 1.2.0
[error] 	    +- com.typesafe.play:twirl-api_2.13:1.6.1             (depends on 2.2.0)
[error] 	    +- com.typesafe.play:play-ws-standalone-xml_2.13:2.1.10 (depends on 1.2.0)
[error] 

```

I also opened this issue in the Silhouette repo: https://github.com/honeycomb-cheesecake/play-silhouette/issues/39

Does anybody know about the status of silhouette?

@mkurz I think you tried to contact Simon over play 2.9 support?

**Comment by MathisGuillet1** [comment]
I think you can manage such issues with "libraryDependencySchemes".
By the way, I faced issues while running silhouette with play 2.9.0-RC3.

I think the changes on "Results" class have impacts on silhouette library.
I don't know what it is worth but I raised a draft PR about upgrading silhouette to play 2.9, see https://github.com/honeycomb-cheesecake/play-silhouette/pull/40

**Reply by mkurz** [reply]
I once more pinged Simon, if he does not continue supporting the library, maybe he can transfer it over to us (play) and I can assign some maintainers. In worst case we can just fork it.

**Reply by mkurz** [reply]
@nemoo @MathisGuillet1 Simon answered, we will transfer the repo in the playframework GitHub organization, are you interested to become maintainers of that repo?

**Reply by MathisGuillet1** [reply]
Yes I'm interested, I will try to help on silhouette

**Reply by mkurz** [reply]
@MathisGuillet1 perfect, I will add you as maintainer when the transfer is done.

**Reply by MathisGuillet1** [reply]
Do you think I can switch the organization from "io.github.honeycomb-cheesecake" to "org.playframework.silhouette" now that the transfer is done ?

**Reply by mkurz** [reply]
Yes I was thinking about that... That does make sense. However, this will look like it's an official Play module then, but this really depends on you to maintain it. We can not pull in every project and then expect me or some other core maintainer to maintain it... So, if we do that you have to spend time on it, otherwise it will just get outdated along the way.

What I can do (the next days) I can set up the whole project with correct GitHub action workflows, correct publishing, correct groupId, etc.
Afterwards you can do your thing. Releasing will then be only pushing a new tag.

**Reply by MathisGuillet1** [reply]
Yes I understand. I will do my best to maintain this project (and try to find other maintainers).
I think it can live as a standalone project as it is not part of play at all.
I hope you can advise on a new organization/groupId to release in standalone mode.
Thanks for your help to set up the workflows.

**Reply by mkurz** [reply]
btw, I already opened a PR to make scala-steward visit the repo: https://github.com/VirtusLab/scala-steward-repos/pull/302
